mobile/src/pages/Login.tsx

```diff
@@ -57,16 +58,20 @@ export default function Login({ navigation }) {
           <Button
             name="Login"
             className="mb-10"
+            disabled={isPending}
             onPress={() => {
+              setIsPending(true);
               login(username, password)
                 .then(async (data) => {
                   if (data) {
                     storeAuthSession(data);
                     await navigation.navigate("FeedPage");
+                    setIsPending(false);
```

Comment:
I think this should go above `navigation.navigate()` because the component might be destroyed when that happens.